### PR TITLE
Move all windows entries to a win-chocolatey.yaml file.

### PR DESCRIPTION
**Pull request by seanyen**
This is a refactor for upstream preparation.

### CODE REVIEW

**Comment by seanyen** [comment]
This change is smoke verified here: https://ros-win.visualstudio.com/ros-win/_build/results?buildId=2929

**Review by kejxu — APPROVED** [review]
lgtm

File: rosdep/python.yaml [hunk 1]
```diff
@@ -4643,4 +4537,4 @@ yapf:
     yakkety:
       pip:
         packages: [yapf]
-    zesty: [yapf]
+    zesty: [yapf]
```
Comment (kejxu) [inline rosdep/python.yaml]: maybe add a new line at the end of this line?

File: rosdep/win-chocolatey.yaml [hunk 2]
```diff
@@ -1,30 +1,338 @@
...
+vcpython:
+  windows: [vcpython27]
+wxpython:
+  windows:
+    pip:
+      package: [wxPython]
+yaml-cpp:
+  windows: [yaml-cpp]
+zlib:
+  windows: [zlib]
```
Comment (kejxu) [inline rosdep/win-chocolatey.yaml]: maybe add a new line at the end of this line?

**Pull request merged**